src/relay/pass/fuse_ops.cc

```diff
@@ -225,6 +225,7 @@ class IndexedForwardGraph::Creator : private ExprVisitor {
     }
 
     node->pattern = op_pattern;
+    this->Update(call->op, nullptr, kOpaque);
```

Comment:
why?

Reply:
call->op might be a nested expression. in such a case we have to recurse into it as well.